exchanges/stream/websocket.go

```diff
@@ -488,15 +509,26 @@ func (w *Websocket) trafficMonitor() {
 				w.setTrafficMonitorRunning(false)
 				w.Wg.Done()
 				return
-			case <-w.TrafficAlert:
-				if !trafficTimer.Stop() {
-					select {
-					case <-trafficTimer.C:
-					default:
+			case t := <-w.TrafficAlert:
+				if w.AuthConn != nil && w.AuthConn.GetURL() == t {
+					if !trafficAuthTimer.Stop() {
+						select {
+						case <-trafficAuthTimer.C:
+						default:
+						}
 					}
+					w.setConnectedStatus(true)
+					trafficAuthTimer.Reset(w.trafficTimeout)
+				} else if w.Conn != nil && w.Conn.GetURL() == t {
+					if !trafficTimer.Stop() {
+						select {
+						case <-trafficTimer.C:
+						default:
+						}
```

Comment:
Can be changed to: 
```go
<-trafficTimer.C
```

Reply:
But it;s already  <-trafficTimer.C  :)

Reply:
Yeah, we can remove the select case. As it shouldn't be needed.